docker-compose.yml

```diff
@@ -1,36 +1,91 @@
 services:
```

Comment:
There is a problem in this file, probably a conflict which is not merged. You should resolve the conflict and merge the result before continue; VS Code should help with this.